cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/BaseController.java

```diff
@@ -31,6 +31,32 @@
         UpdateController.class})
 public class BaseController implements Callable<Integer> {
 
+    private static class LoggingOptions {
+        @CommandLine.Option(names = {"-q", "--quiet"}, description = "Reduce log verbosity and print errors only.")
```

Comment:
You can use a static import like: 

import static picocli.CommandLine.Option; 

then you can just write: 
Option(names = {"-q", "--quiet"}, description = "Reduce log verbosity and print errors only.")

Reply:
True. I hardly ever use those static imports, but the rest of the picocli stuff is imported that way, too, so why not, right?